telemetry_device/Pipeline.cs

```diff
@@ -71,12 +71,17 @@ private void InitializeIcdDictionary()
         private bool FilterPacket(Packet packet)
```

Comment:
change name of func to `IsPacketAppropriate`. Know that when you write func that return bool, the name of func start with Is.